apps/www/src/lib/plate/demo/values/slashRules.ts

```diff
@@ -0,0 +1,61 @@
+import { TComboboxItem } from '@udecode/plate-combobox';
+import { PlateEditor, toggleNodeType } from '@udecode/plate-core';
+import { ELEMENT_H1, ELEMENT_H2, ELEMENT_H3 } from '@udecode/plate-heading';
+import { ListStyleType, toggleIndentList } from '@udecode/plate-indent-list';
+import { focusEditor } from '@udecode/slate-react';
+
+export const SLASH_ENABLED: TComboboxItem[] = [
+  { key: '0', text: 'Heading 1' },
+  { key: '1', text: 'Heading 2' },
+  { key: '2', text: 'Heading 3' },
+  {
+    key: '3',
+    text: 'Bulleted list',
+  },
+  {
+    key: '4',
+    text: 'Numbered list',
+  },
+];
+
+export const SLASH_TRIGGER = [
+  {
+    matchText: 'Heading 1',
```

Comment:
Should these just be the `ELEMENT_CONST_VALUE`?

Reply:
Thanks, it's better.

Reply:
What's better? Wouldn't it be better to use the elements const rather than pulling out the string for matching?